src/Microsoft.DotNet.Tools.Builder/CompileContext.cs

```diff
@@ -125,19 +133,25 @@ private bool NeedsRebuilding(ProjectContext project, ProjectDependenciesFacade d
 
             if (!newInputs.Any())
             {
-                Reporter.Output.WriteLine($"\nProject {project.ProjectName()} was previously compiled. Skipping compilation.");
+                Reporter.Output.WriteLine($"Project {project.GetDisplayName()} was previously compiled. Skipping compilation.");
```

Comment:
I thought about it and it does not matter, because the file system does not have time zone information (as far as I know). So using GetLastWriteTimeUtc just converts GetLastWriteTime to UTC, with not much effect.